### PR TITLE
Remove diacritics when filter vaults members

### PR DESCRIPTION
**Pull request by Fraggle**
## Description

Remove diactritics when filtering vaults members.

https://github.com/user-attachments/assets/8c1d1453-e753-423d-852f-0401a00e89a7

## Risk

None

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/components/vaults/ManageMembersModal.tsx [hunk 1]
```diff
@@ -70,6 +71,15 @@ export function ManageMembersModal({
           </DataTable.CellContent>
         ),
         accessorFn: (row: RowData) => row.name,
+        filterFn: (row: Row<RowData>, columnId: string, filterValue: any) => {
+          if (!filterValue) {
+            return true;
+          }
+
+          return removeDiacritics(row.original.name)
```
Comment (PopDaph) [inline front/components/vaults/ManageMembersModal.tsx:79]: NIT: I think you can do `row.getValue("name")`

File: front/components/vaults/VaultMembers.tsx [hunk 2]
```diff
@@ -33,6 +33,15 @@ const tableColumns = [
       </DataTable.CellContent>
     ),
     accessorFn: (row: RowData) => row.name,
+    filterFn: (row: Row<RowData>, columnId: string, filterValue: any) => {
+      if (!filterValue) {
+        return true;
+      }
+
+      return removeDiacritics(row.original.name)
```
Comment (PopDaph) [inline front/components/vaults/VaultMembers.tsx:41]: NIT: Same here, I think you can do `row.getValue("name")`

**Review by PopDaph — APPROVED** [review]
LGTM! 

For big workspaces I'm not sure we'll be able to load those modal without pagination 😬

File: front/components/vaults/VaultMembers.tsx (hunk 2 above)
Comment (Fraggle) [reply front/components/vaults/VaultMembers.tsx:41]: thanks ! I was looking for that to make the function easily re-usable.

**Pull request merged**